Static/Row.swift

```diff
@@ -148,8 +148,8 @@ public struct Row: Hashable, Equatable {
         return cellClass.description()
     }
 
-    public var hashValue: Int {
-        return uuid.hashValue
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(uuid)
```

Comment:
Yay `Hasher`!

Reply:
Actually, I wanted to ask you about this API change if you are familiar. 
- Actually is this backward incompatible change? What would happen if a user of this library are using this `hashValue` on its app? Or, actually `hashValue` will be generated by the compiler automatically when compiling?

Reply:
I don't think this is necessary "backward" compatible, since I don't think we declare that hash values will be stable between versions 😅 

This change will still allow for the `hashValue` to be retrieved from `Row`s, so I think this is a totally valid change. As far as I understand it, `hashValue` will still be generated at runtime, but this will allow for calculations of the hash values to be generated by other implementations of `Hasher`s.

👍

Reply:
Thanks for explaining, Eli! I also found a good description at Hashable Swift Evolution proposal page 😄 
[SE-0206 Hashable Enhancements Source compatibility](https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md#source-compatibility)
> Code written for Swift 4.2 or later should conform to Hashable by implementing hash(into:). The compiler will then complete the conformance with a suitable hashValue definition:

Whe a type conforms to Hasahble, and `hash(into:)` is implemented, `hashValue` will be synthesized by the compiler automatically.

It was great learning Hashable 😄